client/lib/wp/sync-handler/whitelist-handler.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Module dependencies
+ */
+
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:sync-handler:whitelist' );
+
+const whitelist = [
+	'^/batch',
+	'^/me',
+	'^/products$',
+	'^/read',
+	'^/sites/.+/'
```

Comment:
Yes, we can. In fact I've been adding endpoints in the development process. So let me know if you have a specific list to set here.